docs/docs/01-welcome/02-install.md

```diff
@@ -78,7 +78,7 @@ In this case, use sudo before `curl` and before `bash`:
 sudo curl https://get.ignite.com/cli | sudo bash
 ```
 
-## Upgrading your Ignite CLI installation
+## Upgrading your Ignite CLI installation{#upgrade}
```

Comment:
```suggestion
## Upgrading your Ignite CLI installation {#upgrade}
```

Reply:
It would be great to update it for the other versions that still have no space in between so they are consistent:
- docs/versioned_docs/version-v0.25.2/guide/01-install.md
- docs/versioned_docs/version-v0.26.1/01-welcome/02-install.md
- docs/versioned_docs/version-v0.27.1/01-welcome/02-install.md

Thank you 🙏

Reply:
Done! Sorry, I didn't catch it the first time.

Reply:
That was fast!

Reply:
;D